web/app_options.js

```diff
@@ -105,6 +105,11 @@ const defaultOptions = {
     value: false,
     kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
   },
+  printResolution: {
```

Comment:
Please move this below to https://github.com/mozilla/pdf.js/blob/1421b2f2056ae6dac564469374ff43342a3fb891/web/app_options.js#L224-L236 instead, since the `printResolution` doesn't make sense for e.g. `MOZCENTRAL` builds.

Reply:
Did that. Thanks.